src/ResourceManager/Sql/Commands.Sql/help/Get-AzureRmSqlServerDnsAlias.md

```diff
@@ -0,0 +1,138 @@
+---
+external help file: Microsoft.Azure.Commands.Sql.dll-Help.xml
+Module Name: AzureRM.Sql
+online version: 
+schema: 2.0.0
+---
+
+# Get-AzureRmSqlServerDnsAlias
+
+## SYNOPSIS
+{{Fill in the Synopsis}}
```

Comment:
@maboja-msft please make sure that these templates (synopsis, description, etc.) are filled out for all of the markdown files you are adding